packages/dom/src/dom/document-has-selection.js

```diff
@@ -15,8 +15,9 @@ import documentHasTextSelection from './document-has-text-selection';
  */
 export default function documentHasSelection( doc ) {
 	return (
-		isTextField( doc.activeElement ) ||
-		isNumberInput( doc.activeElement ) ||
-		documentHasTextSelection( doc )
+		!! doc.activeElement &&
```

Comment:
If a document has no `activeElement` then it cannot have a selection. Furthermore, `isTextField` and `isNumberInput` access the passed in element completely unguarded, so the assumption here was always that `activeElement` was defined.